GitCommands/Config/ConfigFile.cs

```diff
@@ -296,8 +292,9 @@ private class ConfigFileParser
             private ConfigFile _configFile;
             private string _fileContent;
             private IConfigSection _section = null;
-            private string FileName { get { return _configFile._fileName; } }
             private string _key = null;
+            private string FileName => _configFile.FileName;
```

Comment:
:memo: Conversion to expression-bodied members should be sent as a separate pull request